controllers/admin/self-managed/UpdatePageVersionChoiceController.php

```diff
@@ -256,4 +233,41 @@ public function submit(): JsonResponse
         /* we dont check again because the button is only accessible if check are ok */
         return AjaxResponseBuilder::nextRouteResponse(Routes::UPDATE_STEP_UPDATE_OPTIONS);
     }
+
+    public function coreTemperedFilesDialog(): JsonResponse
+    {
+        return AjaxResponseBuilder::hydrationResponse(
+            PageSelectors::DIALOG_PARENT_ID,
+            $this->getTemperedFilesDialog([
+                'title' => $this->upgradeContainer->getTranslator()->trans('List of core alterations'),
+                'message' => $this->upgradeContainer->getTranslator()->trans('Some core files have been altered, customization made on these files will be lost during the update.'),
```

Comment:
there is already the wording present here https://github.com/PrestaShop/autoupgrade/blob/97045a8b72d88f32c666a645ec0f6e9b2773d81a/classes/UpgradeSelfCheck.php#L375, same for theme